SingularityBase/src/main/java/com/hubspot/singularity/api/SingularityRunNowRequest.java

```diff
@@ -67,6 +82,11 @@ public SingularityRunNowRequest(@JsonProperty("message") Optional<String> messag
     return resources;
   }
 
+  @ApiModelProperty(required=false, value="Additional artifacts to download for this run")
+  public Optional<List<SingularityMesosArtifact>> getExtraArtifacts() {
```

Comment:
Instead of passing Optional lists around, let’s just default it to an empty list if the constructor arg is null. Same for the other objects that use this. Will save a lot of isPresent being throw around